sycl/include/CL/sycl/detail/context_impl.hpp

```diff
@@ -121,6 +121,21 @@ class context_impl {
   ///
   /// @return a pointer to USM dispatcher.
   std::shared_ptr<usm::USMDispatcher> getUSMDispatch() const;
+
+  /// Returns device library programs: in contrast to user programs, which are a
```

Comment:
Perhaps worth keeping the doxygen style comments for this function e.g. @return, just a suggestion though!

Reply:
Changed the wording and added a `@return` line.